extensions/objects/templates/svg-filename-editor.component.ts

```diff
@@ -709,11 +721,10 @@ angular.module('oppia').component('svgFilenameEditor', {
       };
 
       var drawQuadraticCurve = function() {
-        var size = ctrl.fabricjsOptions.size;
         var curve = new fabric.Path('M 40 40 Q 95, 100, 150, 40', {
           stroke: ctrl.fabricjsOptions.stroke,
           fill: ctrl.fabricjsOptions.fill,
-          strokeWidth: parseInt(size.substring(0, size.length - 2)),
+          strokeWidth: getStrokeWidth(),
           objectCaching: false,
           selectable: false
         });
```

Comment:
Avoid "magic numbers" passed to `createBezierControlPoints(..)` below. Assign it to descriptive variables and use those.

Reply:
Done.